src/io/flutter/FlutterErrorReportSubmitter.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2016 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+package io.flutter;
+
+import com.intellij.ide.DataManager;
+import com.intellij.ide.plugins.IdeaPluginDescriptor;
+import com.intellij.ide.plugins.PluginManager;
+import com.intellij.ide.scratch.ScratchRootType;
+import com.intellij.lang.Language;
+import com.intellij.openapi.actionSystem.DataContext;
+import com.intellij.openapi.actionSystem.PlatformDataKeys;
+import com.intellij.openapi.diagnostic.ErrorReportSubmitter;
+import com.intellij.openapi.diagnostic.IdeaLoggingEvent;
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.diagnostic.SubmittedReportInfo;
+import com.intellij.openapi.extensions.PluginId;
+import com.intellij.openapi.fileEditor.OpenFileDescriptor;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.project.ProjectManager;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.util.Consumer;
+import io.flutter.sdk.FlutterSdk;
+import org.apache.commons.compress.utils.Charsets;
+import org.apache.commons.compress.utils.IOUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.awt.*;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class FlutterErrorReportSubmitter extends ErrorReportSubmitter {
+  private static final Logger LOG = Logger.getInstance(FlutterErrorReportSubmitter.class);
+
+  @NotNull
+  @Override
+  public String getReportActionText() {
+    return "Create Flutter Bug Report";
+  }
+
+  @SuppressWarnings("deprecation")
+  @Override
+  public void submitAsync(IdeaLoggingEvent[] events,
```

Comment:
Looking at the docs for `ErrorReportSubmitter`, this is deprecated in favor of `submit(IdeaLoggingEvent[], String, Component, Consumer)`?

Also, nit: I'd copy the annotations over too so that we know (for example) that `events` is certain not to be null.